package.json

```diff
@@ -117,15 +117,148 @@
                     "description": "How much to extend neovim viewport on each side beyond vscode editor size. Increase if you are not seeing decorations or highlights on the edges of the editor."
                 },
                 "vscode-neovim.useCtrlKeysForNormalMode": {
+                    "deprecationMessage": "Use vscode-neovim.ctrlKeysForNormalMode instead (Set it to empty)",
                     "type": "boolean",
                     "default": true,
                     "description": "Use Ctrl keys in normal/visual/etc...(except insert) modes"
                 },
                 "vscode-neovim.useCtrlKeysForInsertMode": {
+                    "deprecationMessage": "Use vscode-neovim.ctrlKeysForInsertMode instead (Set it to empty)",
                     "type": "boolean",
                     "default": true,
                     "description": "Use Ctrl keys in insert mode"
                 },
+                "vscode-neovim.ctrlKeysForNormalMode": {
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "type": "string",
+                        "enum": [
```

Comment:
There are a lot of other possibilities here, like :1234567890,.\ and space/pgup/pgdown/esc. Do you think it is worth it to include more?

Reply:
We only need to define the relatively common ones. This configuration option itself is for auxiliary purposes, so it’s fine to have a little more.